csv_parser/csv_parser.py

```diff
@@ -56,10 +56,11 @@ def get_params_from_sheet(sheet):
         # Save all the column numbers that have 'Périmètre' in their name
         perimeter_columns = [i for i, x in enumerate(perimeter_row) if 'Périmètre' in str(x)]
         # Computing number of rows in table
-        # rows = df.iloc[7:, 0]
-        # Simply remove initial rows & total row
-        # ToDo: be more resilient to nan & \xa0 in full_df.iloc[8:,0] and compute nb with count?
-        rows = full_df.shape[0] - 8 - 1
+        # Find the row number of the first table header ('ID')
+        id_index = (full_df[0] == 'ID').idxmax()
```

Comment:
```suggestion
        id_index = (full_df[0] == 'ID').idxmin()
``` 
plutôt pour avoir le 1er non ?

Reply:
Vu qu'on n'a qu'une seule colonne "ID'" ça devrait rien changer, mais je suis d'accord que c'est plus raisonnable de point de vue logique de prendre la première occurrence